exercises/practice/hello-world/src/hello-world.CBL

```diff
@@ -5,7 +5,6 @@
         WORKING-STORAGE SECTION.
         01 WS-RESULT PIC X(13).
         PROCEDURE DIVISION.
-            GOBACK.
```

Comment:
@KTSnowy, according to https://github.com/openmainframeproject/cobol-check/issues/223 the `GOBACK` is not needed and the test is running fine without it. So, I removed it.

Reply:
That's good to know, I'm glad that it's still working without the `GOBACK`

Could you check something for me, does cobol-check work with free format and lowercase keywords?
You might have to pass the `-free` command line option to gnuCOBOL.
```cobol
*> Sample COBOL program
identification division.
  program-id. hello.

data division.
  working-storage section.
    01 ws-result pic x(13).

procedure division.
  hello-world.
  move "Goodbye, Mars!" to ws-result.
stop run.
            
```

Reply:
Lower case letters are supported in Cobol check, but free format is not. You can make an issue on that, if it's something you need.